intelligence-services/latest/using/index.md

```diff
@@ -48,7 +48,9 @@ Follow these steps to use the default (i.e. out-of-the-box) configuration of Int
         aiFeatures, aiLabels, aiTextract, aiSpeechToText, webvtt, aiPiiEntities 
         ```
 
-        > **Note:** If you leave the text field empty, all three of the default renditions will be requested. The default renditions are `aiFeatures` for Amazon Comprehend, `aiLabels` for Amazon Rekognition of images, and `aiTextract` for Amazon Textract.
+        > **Note:** If you leave the text field empty all of the default renditions will be requested.
+
+        The default renditions are, **aiLabels** -> AI Labels, **aiTextract** -> AI Text Lines, aiFeatures -> AI Creative Works, AI Dates, AI Events, AI Organizations, AI People, AI Places, AI Products, AI Quantities, AI Things, **aiPiiEntities** -> AI PII Entities, **aiSpeechToText** -> AI Transcript, **webVTT** -> n/a.  
```

Comment:
As per my previous comment, this should be a list of renditions.

As per DOCS-6242, there should be a new step between 7 and 8 where user chooses to apply zero or more specific AI aspects (for specific AI renditions) in order to extract the metadata and set as Alfresco content properties. 

This mapping should be part of the new step + a new screenshot would help (similar to Custom AI aspects).

Reply:
G'day
@montgolfiere 
The list has been done. The other changes will be done as part of  DOCS-6242 which is not a part of this original JIRA.
Thanks for your help on this.